_post/chitranshi135.txt

```diff
@@ -0,0 +1,2 @@
+NAME : CHITRANSHI SRIVASTAVA
+Introduction to open source
```

Comment:
PTAL(please take a look) over `sample.txt` file under `_post` folder and then add file accordingly.